docs/source/cn/tensor.py

```diff
@@ -47,5 +63,1343 @@
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.dim,
+    r"""
+    Tensor.dim() → int
+
+    返回self张量的维数。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.element_size,
+    r"""
+    Tensor.element_size() → int
+
+    返回单个元素的字节大小。
+
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.fill_,
+    r"""
+    Tensor.fill_(value) → Tensor
+
+    用指定的值填充 self 张量。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.get_device,
+    r"""
+    Tensor.get_device() -> Device ordinal (Integer)
+
+    对于 CUDA 张量，该函数返回张量所在 GPU 的设备序号。对于 CPU 张量，会产生一个错误。
+
+
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.mul_,
+    r"""Tensor.mul_(value) -> Tensor
+
+    :func:`oneflow.Tensor.mul` 的原版本。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.div_,
+    r"""Tensor.div_(value) -> Tensor
+    :func:`oneflow.Tensor.div` 的原版本。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.sub_,
+    r"""Tensor.sub_(value) -> Tensor
+    :func:`oneflow.Tensor.sub` 的原版本。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.nelement,
+    r"""
+    Tensor.nelement() → int
+
+    numel() 的别名。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.floor_,
+    r"""
+    :func:`oneflow.floor` 的原版本。
+
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.normal_,
+    """
+    normal_(mean=0, std=1, *, generator=None) -> Tensor
+
+    用正态分布的元素样本填充 :attr:`self` 张量，参数为 :attr:`mean` 和 :attr:`std`。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.numpy,
+    r"""
+    Tensor.numpy() → numpy.ndarray
+
+    将 self 张量作为一个 NumPy ndarray 返回。这个张量和返回的 ndarray 共享相同的底层存储。对 self 张量的变化将反映在 ndarray 中，反之亦然。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.add_,
+    r"""
+    :func:`oneflow.Tensor.add` 的原版本。
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.size,
+    r"""
+    该接口与 PyTorch 一致。
+
+    返回 self 张量的大小。如果没有指定 dim ，返回值是 oneflow.Size ，是 tuple 的子类。如果指定了 dim ，返回一个持有该维度大小的 int 。
+
+    参数:
+        - **idx** (int, optional) - 用于检索尺寸的维度。
+
+
+    """,
+)
+
+reset_docstr(
+    oneflow.Tensor.to_local,
+    r"""返回一个全局张量的局部张量。
+
+
+    参数:
+        - **input** (Tensor) - 输入张量。
+
+    例如:
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> import numpy as np
+        >>> np_arr = np.array([0.5, 0.6, 0.7]).astype(np.float32)
+        >>> input = flow.tensor(np_arr, dtype=flow.float32)
+        >>> placement = flow.placement("cpu", ranks=[0])
+        >>> global_tensor = input.to_global(placement, [flow.sbp.split(0)])
+        >>> global_tensor.to_local()
+        tensor([0.5000, 0.6000, 0.7000], dtype=oneflow.float32)
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.uniform_,
+    r"""
+    Tensor.uniform_(from=0, to=1) → Tensor
+
+    用从连续均匀分布中采样的数字填充 self 张量。
+
+    .. math::
+        P(x)=1/(to-from)
+
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.copy_,
+    r"""
+    该接口与 PyTorch 一致。
+
+    Tensor.copy_(src, non_blocking=False) → Tensor
+
+    将 src 中的元素复制到 self 张量中，并返回 self 。
+
+    src 张量必须可以与 self 张量一起广播。它可能是不同的数据类型，或者停留在不同的设备上。
```

Comment:
```suggestion
    src 张量必须可以与 self 张量一起广播。它可以是不同的数据类型，或者位于不同的设备上。
```
感觉这些词都可以翻译的更加人性化一些（或者更加意译一些），不必太纠结于词最基本的意思